extensions/gdx-tools/src/com/badlogic/gdx/tiledmappacker/TiledMapPacker.java

```diff
@@ -480,12 +480,11 @@ public static void main (String[] args) {
 		}
 
 		TiledMapPacker packer = new TiledMapPacker(packerSettings);
```

Comment:
The documentation of `TiledMapPacker#main()` (lines 128-134) needs to be adapted as well.